pkg/mountutil/mountutil_windows_test.go

```diff
@@ -38,7 +38,7 @@ func TestParseVolumeOptions(t *testing.T) {
 			vType:   "bind",
 			src:     "dummy",
 			optsRaw: "rw",
-			wants:   []string{},
```

Comment:
Either the code or the test should be changed.

I am opting for changing the test.